tests/integration/security/fuzz/fuzz_test.go

```diff
@@ -104,13 +104,8 @@ func deploy(t framework.TestContext, name, ns, yaml string) {
 func waitService(t framework.TestContext, name, ns string) {
 	if _, _, err := kube.WaitUntilServiceEndpointsAreReady(t.Clusters().Default().Kube(), ns, name); err != nil {
 		t.Fatalf("Wait for service %s failed: %v", name, err)
-		if name == apacheServer || name == nginxServer || name == tomcatServer {
```

Comment:
This doesn't seem related to the PR?

Reply:
after Fatalf the function exits; this would never run. new Linter picks it up. The code is a duplicate of what is on L105 anyways